examples/amoeba/in.ubiquitin

```diff
@@ -34,6 +34,15 @@ pair_coeff      * * amoeba_ubiquitin.prm amoeba_ubiquitin.key
 
 special_bonds   lj/coul 0.5 0.5 0.5 one/five yes
 
+# setup force components this way so can dump them (AMOEBA or HIPPO also needs them for now)
```

Comment:
There is no reference to those fixes in the source code anywhere. Instead pair style amoeba creates an instance of fix STORE/PERATOM internally.

Reply:
Removed this line in the input script. This is a leftover from older versions.